### ISSUE TITLE
original MNIST experiment

### ISSUE THREAD

**Issue by DaNavUser** [DaNavUser/NIPS_Experiments#39]
3 fully-connected hidden layers with 100 activations each.
Each hidden layer computes y = g(Wu+b) with sigmoid
nonlinearity, and the weights W initialized to small random Gaussian values. The last hidden layer is followed
by a fully-connected layer with 10 activations (one per
class) and cross-entropy loss. We trained the network for
50000 steps, with 60 examples per mini-batch. We added
Batch Normalization to each hidden layer of the network,
as in Sec. 3.1. We were interested in the comparison between the baseline and batch-normalized networks, rather
than achieving the state of the art performance on MNIST
(which the described architecture does not).
